config/default.json

```diff
@@ -2,6 +2,7 @@
   "providerUrl": "http://127.0.0.1:8545",
   "network": "ganache",
   "gasLimit": 6300000,
```

Comment:
Please rename "gasLimit" to "gasLimit.deployment"

Reply:
done at 3f39eb4